app/src/main/java/de/westnordost/streetcomplete/data/quest/QuestTypeRegistry.kt

```diff
@@ -9,4 +9,4 @@ import de.westnordost.streetcomplete.data.ObjectTypeRegistry
  * It is also used to define a (display) order of the quest types and to assign an ordinal to each
  * quest type for serialization.
  */
-class QuestTypeRegistry(ordinalsAndEntries: List<Pair<Int, QuestType>>) : ObjectTypeRegistry<QuestType>(ordinalsAndEntries)
+class QuestTypeRegistry(val ordinalsAndEntries: List<Pair<Int, QuestType>>) : ObjectTypeRegistry<QuestType>(ordinalsAndEntries)
```

Comment:
No need to expose this this parameter. `ObjectTypeRegistry<T>` already inherits from `AbstractList<T>`, so you can access the contents directly.